db/db_bloom_filter_test.cc

```diff
@@ -3113,6 +3113,382 @@ TEST_F(DBBloomFilterTest, SeekForPrevWithPartitionedFilters) {
   it.reset();
 }
 
+namespace {
+class BackwardBytewiseComparator : public Comparator {
+ public:
+  const char* Name() const override { return "BackwardBytewiseComparator"; }
+
+  int Compare(const Slice& a, const Slice& b) const override {
+    size_t min_size = std::min(a.size(), b.size());
+    const char* a_end = a.data() + a.size();
+    const char* b_end = b.data() + b.size();
+    for (size_t i = 1; i <= min_size; ++i) {
+      if (a_end[-i] == b_end[-i]) {
```

Comment:
`i` is unsigned (`size_t`) => `-i` is also unsigned ("-1" becomes 128, or 255 I believe)